src/test/java/MathematicalCalculationsTest.java

```diff
@@ -0,0 +1,12 @@
+import Exceptions.DivisionByZero;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MathematicalCalculationsTest {
+    @Test
+    void testCalculations() throws DivisionByZero {
+        String expression = "3*(3+4)";
+        assertEquals(21, MathematicalCalculations.Calculations(ExpressionParser.parseExpressionInReversePolandNotation(expression)));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.